src/adapters/git.ts

```diff
@@ -70,9 +70,7 @@ abstract class GitAdapter implements IRepoAdapter {
   }
 
   public async pushRepo(repo: IRepo, force: boolean): Promise<void> {
-    const options = {
-      '--force': force,
-    };
+    const options = force ? ['--force'] : undefined;
```

Comment:
Link to prior discussion on this: https://github.com/NerdWalletOSS/shepherd/pull/353#discussion_r552390311